api/src/opentrons/protocol_api/labware.py

```diff
@@ -1000,7 +1036,7 @@ def return_tips(self, start_well: Well, num_channels: int = 1) -> None:
 
     @requires_version(2, 0)
     def reset(self) -> None:
-        """Reset all tips in a tip rack.
+        """Reset tip tracking for a tip rack.
```

Comment:
Maybe something like:

```suggestion
        """Reset the tracking for which tips have been used.
```

Since I don't know if it's otherwise clear what we mean by "tip tracking."

Reply:
Added a lot of explanation. Hopefully not too much!